libs/platform/src/lib/components/split-menu-button/split-menu-button.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="fd-button-split" role="group" aria-label="button-split">
+    <button
+        fd-button
+        [attr.id]="id"
+        [attr.name]="name"
+        aria-label="buttonLabel"
+        [disabled]="disabled"
+        [fdType]="type"
+        [glyph]="icon"
+        [compact]="contentDensity === 'compact'"
+        (click)="primarButtonClicked($event)"
+    >
+        {{ buttonLabel }}
+    </button>
+    <button
+        fd-button
+        [attr.id]="id"
```

Comment:
The secondary button needs a different `id` than the primary. In your code you should create a new variable, say `idSecond`, and then programmatically compose the `idSecond` by taking `id` and appending a string to it, maybe something like, `-secondary`.